youtube_dl/extractor/mgtv.py

```diff
@@ -82,14 +83,17 @@ def _real_extract(self, url):
                 'http_headers': {
                     'Referer': url,
                 },
+                'format_note': stream.get('name'),
             })
         self._sort_formats(formats)
 
+        desc = info.get('desc')
         return {
             'id': video_id,
             'title': title,
             'formats': formats,
-            'description': info.get('desc'),
+            'description': desc,
+            'alt_title': desc,
```

Comment:
Description can be used just fine in output template. Revert.

Reply:
Will do, but description is not mentioned in https://github.com/ytdl-org/youtube-dl#output-template?

Reply:
It's not cause descriptions are usually multiline and using them will likely mess filenames. But technically nothing stops from using them in output template. As well as any other meta field available in info dict.